indico/modules/events/contributions/templates/management/_subcontribution_list.html

```diff
@@ -0,0 +1,110 @@
+{% macro render_subcontribution_list(event, contrib) %}
+    <div id="subcontribution-list" class="i-form">
```

Comment:
It's not a form (but has a form somewhere inside and you cannot nest forms). Doesn't look like a good idea though.. What i-form styling do you need?